packages/presets/near-operation-file/src/utils.ts

```diff
@@ -39,12 +39,18 @@ export function extractExternalFragmentsInUse(documentNode: DocumentNode | Fragm
 
 export function fixLocalFile(path: string): string {
   if (!path.startsWith('..') && !path.startsWith('&&')) {
```

Comment:
When `path` could start with `&&`? IMO this condition doesn't make sense.

Reply:
Can be removed, added it for tests